common/djangoapps/terrain/ui_helpers.py

```diff
@@ -43,8 +43,10 @@
 
     # Content - Outline
     # Note that calling your org, course number, or display name, 'course' will mess this up
+    # Waiting for the factories to load - they load everything else through dependencies.
+    # Specifying them individually as they are independent from each other.
     re.compile(r'^Course Outline \|'): [
-        "js/base", "js/models/course", "js/models/location", "js/models/section"],
+        "js/base", "factories/base.js", "factories/course.js", "factories/outline.js"],
```

Comment:
@andy-armstrong When I took out js/base I got errors when loading that page. Putting it back in. Failures were arbitrarily in any of the factories file, i.e. one of these:
```
AssertionError: Error loading dependencies: type=scripterror modules=[u'factories/base.js']
AssertionError: Error loading dependencies: type=scripterror modules=[u'factories/course.js']
AssertionError: Error loading dependencies: type=scripterror modules=[u'factories/outline.js']
```